webpack.config.js

```diff
@@ -143,6 +144,14 @@ module.exports = opts => {
         );
       }
 
+      plugins.push(new CopyWebpackPlugin([
```

Comment:
Let's better inline the icon as my wish is to keep the report to be a single html file.